### PR TITLE
update developer and how we work docs

### PR DESCRIPTION
**Pull request by jkueloc**
Fix links for cloudformation and setup.cfg, added nb for pre-commit, inserted link on for-developer @ 'Setting up a local dev..'  to 'Ensuring your work follows...', and moved the testing instruct further down the page.

### CODE REVIEW

File: docs/how-we-work.md [hunk 1]
```diff
@@ -158,9 +158,11 @@ The easiest way to install the site is using [Pipenv](https://pipenv.readthedocs
 Now every time you make a commit in Git the various tools listed in the next
 section will automatically run and report any problems.
 
+n.b. Each time you check out a new copy of this Git repository, run pre-commit install.
```
Comment (acdha) [inline docs/how-we-work.md:161]:
```suggestion
n.b. Each time you check out a new copy of this Git repository, run `pre-commit install`.
```

File: docs/how-we-work.md [hunk 2]
```diff
@@ -158,9 +158,11 @@ The easiest way to install the site is using [Pipenv](https://pipenv.readthedocs
...
+
 #### Configure your editor with helpful tools:
 
-[setup.cfg](setup.cfg) contains configuration for pycodestyle, [isort](https://pypi.org/project/isort/) and flake8.
+[setup.cfg](https://github.com/LibraryOfCongress/concordia/blob/master/setup.cfg) contains configuration for pycodestyle, [isort](https://pypi.org/project/isort/) and flake8.
```
Comment (acdha) [inline docs/how-we-work.md:165]:
```suggestion
[setup.cfg](https://github.com/LibraryOfCongress/concordia/blob/master/setup.cfg) contains configuration for [pycodestyle](https://pypi.org/project/pycodestyle/), [isort](https://pypi.org/project/isort/) and [flake8](https://pypi.org/project/flake8/).
```

**Review by acdha — APPROVED** [review]
I have a couple of minor additions but nothing major

**Comment by jkueloc** [comment]
The UI was kinda confusing. Looked like I needed to commit the recommended changes which of course required another review... if I misunderstood what I should do in the future - please redirect. Thank you for the suggestions!

**Comment by acdha** [comment]
> The UI was kinda confusing. Looked like I needed to commit the recommended changes which of course required another review... if I misunderstood what I should do in the future - please redirect. Thank you for the suggestions!

Looks good to me - once you accepted them it generated a new commit for each one.

**Review by acdha — APPROVED** [review]

**Pull request merged**